### ISSUE TITLE
Improve specificity of email notifications

### ISSUE THREAD

**Issue by GrantASL19** [ASL-19/civicdr-backend#20]
Notifications like the following could be confusing to regular users:

> Update on the platform at 2017-07-21T19:37:03+00:00. Log in to review.

I assume we might want to avoid putting sensitive information in the emails for security reasons, but something like “your ticket has been updated” would be more friendly than referring to “the platform” when regular users might not know what a platform is.

**Comment by MayaAvaz** [comment]
To add to it: Assignment of a new ticket to SP should generate an email to them with a clear message, eg: 'A new ticket has been assigned to you', etc.

**Comment by seamustuohy** [comment]
Addressing this issue in the following branch. https://github.com/ASL-19/civicdr-backend/tree/issue_18_21

The current changes [not only specify the type of event that has occurred, it also specify the number of those events in the reporting period.](https://github.com/ASL-19/civicdr-backend/blob/issue_18_21/utils/mailer.js#L30-L55).

**Comment by seamustuohy** [comment]
Closed by [pull request #24](https://github.com/ASL-19/civicdr-backend/pull/24)

Here is an example of the new email text.
```
The following 9 events have occured on the CiviCDR Platform since Jul 25
15:53 UTC:
 - You were assigned to a ticket.
 - 2 of your tickets were updated.
 - You received 5 messages.
 - A new ticket was created.

You can learn more by logging into the CiviCDR Platform.
```
